packages/reports/addon/components/filter-builders/dimension.hbs

```diff
@@ -2,8 +2,8 @@
 {{#if @isCollapsed}}
   <span class="filter-builder" ...attributes>
     <FilterBuilders::Collapsed
-      @displayName={{this.displayName}}
-      @filter={{this.filter}}
+      @displayName={{@filter.displayName}}
+      @filter={{@filter}}
       @request={{@request}}
       @field={{this.field}}
```

Comment:
Do we still need this?

Reply:
Nope, changed collapsed to use @filter.displayname

Reply:
Can this be removed?

Reply:
Which part @displayName has been removed

Reply:
@holderb5 line `8`

Reply:
No we shouldnt. Deleting it now.